examples/qml_extension_plugin/CMakeLists.txt

```diff
@@ -1,38 +0,0 @@
-# SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
think we need to remove qml_extension_plugin from the README? And from the github test rules?

```bash
$ git grep qml_extension_plugin
.github/workflows/github-cxx-qt-tests.yml:            ctest_args: --exclude-regex '^(example_qml_extension_plugin_test|example_qml_features_test|example_qml_minimal_myobject_test|reuse_lint|cpp_clang_format|.*valgrind)$'
.github/workflows/github-cxx-qt-tests.yml:            ctest_args: --exclude-regex '^(example_qml_extension_plugin_test|example_qml_features_test|example_qml_minimal_myobject_test|reuse_lint|cpp_clang_format|.*valgrind)$'
examples/README.md:  * [Exposing the Rust objects via a QQmlExtensionPlugin](./qml_extension_plugin/plugin/)
```

Reply:
done